### DISCUSSION TITLE
Illegal Instruction error on M1 Mac/Monterey

### DISCUSSION THREAD

**Discussion by m-meller** [The-Wineskin-Project/WineskinServer#283] [Q&A] [answered]
Hello, I have an issue when trying to install a game on macOS Monterey 12.5. My question would be whether it is an issue with the game itself or with the Wineskin/wrapper and Mac being funny. I'm not sure how to proceed with troubleshooting. I'd greatly appreciate any comments.

### General info
Computer: MacBook Air (M1, 2020)
Application: World of Warcraft 32bit custom client (basically version 1.12.1, can be run with DirectX9)
Problem: Program crashes on launch with ILLEGAL_INSTRUCTION error
Engine Version: WS11WineCX64Bit21.2.0 installed with Rosetta 2
Wrapper Version: Wineskin-2.9.1.1

Here are the outputs from the Wineskin logs:

### LastRunWine.log
[LastRunWine.log](https://github.com/Gcenx/WineskinServer/files/9184328/LastRunWine.log)

### LastRunX11.log
[LastRunX11.log](https://github.com/Gcenx/WineskinServer/files/9184329/LastRunX11.log)

### Game crash error
I have also attached the crash log coming from the application. Not sure if relevant but might be useful
[log.txt](https://github.com/Gcenx/WineskinServer/files/9184330/log.txt)

Again, I'd greatly appreciate help.

**Comment by Gcenx** [comment, marked as answer]
From memory I’d seen the same problem when attempting to run WoW Wotlk client on M1, it seems lightly it’s a rosetta2 bug.

I do remember being able to play on my Intel system on macOS Catalina but it would endup lagging after sometime.

**Reply by m-meller** [reply]
Many thanks, for that. Yes, I had no problem running any client on previous systems. At least now I know where to look for updates.
